lib/auth/webauthntypes/webauthn.go

```diff
@@ -405,7 +405,14 @@ type SessionData struct {
 	// An empty value is treated equivalently to "discouraged".
 	UserVerification string `json:"userVerification,omitempty"`
 	// ChallengeExtensions are Teleport extensions that apply to this webauthn session.
-	ChallengeExtensions *mfav1.ChallengeExtensions `json:"challenge_extensions,omitempty"`
+	ChallengeExtensions *ChallengeExtensions `json:"challenge_extensions,omitempty"`
+}
+
+// ChallengeExtensions is a json struct for [mfav1.ChallengeExtensions].
+type ChallengeExtensions struct {
+	Scope                       mfav1.ChallengeScope      `json:"scope,omitempty"`
```

Comment:
Suggestion: Isn't Scope technically required?

```suggestion
	Scope                       mfav1.ChallengeScope      `json:"scope"`
```

Reply:
Does it matter for the encoding?

Reply:
Not for encoding, but it's a way to show that we always expect a scope to be present.